console/src/routes.js

```diff
@@ -54,17 +54,18 @@ const routes = store => {
           cb();
         },
         r => {
-          if (r.code === 'data_api_error') {
+          if (r && r.code === 'data_api_error') {
             dispatch(showErrorNotification('Error', null, r));
           } else {
             dispatch(
               showErrorNotification(
                 'Connection error',
-                'Hasura console is not able to reach your Hasura GraphQL engine instance. Please ensure that your ' +
+                'Hasura console is not able to reach your Hasura CLI instance. Please ensure that your ' +
```

Comment:
Can you elaborate on this change? What if I run console in server mode and there's no CLI involved?

Reply:
Thanks for review. We are only dealing with the cli mode here (its in a `if` condition for CLI), and I guess this error message was mistaken, check for reference: https://github.com/hasura/graphql-engine-mono/blob/main/console/src/routes.js#L57

Reply:
we don't need an api error fallback page in my opinion, I've updated the pr and pr description accordingly